src/modules/ROOT/pages/changelog/202303/06.json

```diff
@@ -75,15 +64,15 @@
     },
     {
       "what": [
-        "filter in progress bar not returning correct data if record id is provided as parameter."
+        "prefilter in progress bar page block not returning any data if records are filtered by record id."
       ],
       "refs": [
         "https://github.com/cortezaproject/corteza/pull/1414"
       ]
     },
     {
       "what": [
-        "being redirected to list of namespaces/pages/modules instead of the previous page after clicking on the 'Save and close' button."
+        "being redirected to list of namespaces/pages/modules instead of the previous page after clicking on the btn:[Save and close] button when creating/editing a namespace/page/module."
```

Comment:
I personally prefer to do `when creating or editing` over `creating/editing`.
I won't fight you over it though